srcpkgs/rocksdb/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'rocksdb'
+pkgname=rocksdb
+version=5.17.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DWITH_TESTS=0 -DUSE_RTTI=0 -DWITH_LZ4=1"
```

Comment:
`option(PORTABLE "build a portable binary" OFF)`
set `-DPORTABLE=1`, we don't want to build with `march=native`

Reply:
will add the flag, hopefully have time tomorrow